src/style.css

```diff
@@ -1 +1,41 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars__star {
+  display: flex;
+  flex-direction: column;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(../src/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+}
+
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+
+.stars--1 .stars__star:nth-child(1) {
+  background-image: url(../src/images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(-n+2) {
+  background-image: url(../src/images/star-active.svg);
+}
+
+.stars--3 .stars__star:nth-child(-n+3) {
+  background-image: url(../src/images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(../src/images/star-active.svg);
+}
+
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url(../src/images/star-active.svg);
+}
```

Comment:
Check how we can combine these selectors (try to use `,`)

https://www.w3schools.com/css/css_combinators.asp